x-pack/plugins/fleet/common/services/agent_policies_helpers.ts

```diff
@@ -44,3 +44,10 @@ function policyHasIntegration(agentPolicy: AgentPolicy, packageName: string) {
 
   return agentPolicy.package_policies?.some((p) => p.package?.name === packageName);
 }
+
+export function getInheritedNamespace(agentPolicies: AgentPolicy[]): string {
+  if (agentPolicies.length === 1) {
+    return agentPolicies[0].namespace;
+  }
+  return 'default';
+}
```

Comment:
Could we write a quick unit test for this? Seems like a good candidate for simple tests.

Reply:
Added one unit test in https://github.com/elastic/kibana/pull/186084/commits/191ad44c2168f7c43352d664c1a44f221d095380